epsilon_transformers/process/MixedStatePresentation.py

```diff
@@ -59,4 +86,48 @@ def path_to_beliefs(self, path: List[int]) -> Float[np.ndarray, "path_length n_s
 
 		assert current_node.path == path, f"{path} is not a valid path for this process"
 		assert len(belief_states) == len(path)
-		return np.stack(belief_states)
\ No newline at end of file
+		return np.stack(belief_states)
+	
+
+	def build_transition_graph(self):
```

Comment:
rename to build_msp_transition_matrix

Reply:
(rename epsilon_transformers/process/MixedStateProcess to epsilon_transformers/process/MixedStateTree